leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java

```diff
@@ -995,12 +1049,12 @@ private void setNewLocation(Address address) {
       currentLocation = new Location(getString(R.string.network_resource));
     }
 
-    double latitude = address.getLatitude();
-    double longitude = address.getLongitude();
+    double lat = address.getLatitude();
```

Comment:
It's better to have the full name. And for the purpose of the PR, it's not necessary to change this methods